src/frontend/js/data/SessionProvider/BaseSessionProvider.tsx

```diff
@@ -58,7 +58,7 @@ const BaseSessionProvider = ({ children }: PropsWithChildren<any>) => {
   }, [queryClient]);
 
   const destroy = useCallback(async () => {
-    await AuthenticationApi!.logout();
+    AuthenticationApi!.logout();
```

Comment:
Maybe it should be tested as well?

Reply:
I just have to update a test to not await the destroy method to check if user state has been updated.